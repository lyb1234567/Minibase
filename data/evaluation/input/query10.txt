Q(x, y, z, u) :- R(x, y, z), S(u, w, t), T(u, r), x > u