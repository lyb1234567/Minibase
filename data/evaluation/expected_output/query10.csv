1, 9, 'adbs', 1
2, 7, 'anlp', 2
4, 2, 'ids', 4
8, 1, 'mlpr', 8
8, 9, 'rl', 8
8, 9, 'ppls', 8
